wallet/src/main/resources/walletdb/migration/V3__wallet_db_add_dlcs.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE IF NOT EXISTS "wallet_dlcs" ("eventId" VARCHAR(254) PRIMARY KEY NOT NULL,"oracleInfo" VARCHAR(254) NOT NULL,"contractInfo" VARCHAR(254) NOT NULL,"timeouts" VARCHAR(254) NOT NULL,"isInitiator" INTEGER NOT NULL,"remoteExtPubKey" VARCHAR(254),"input" INTEGER NOT NULL,"remoteInput" INTEGER,"fundingInputs" VARCHAR(254) NOT NULL,"remoteFundingInputs" VARCHAR(254), "feeRate" VARCHAR(254),"changeSPK" VARCHAR(254) NOT NULL,"remoteChangeSPK" VARCHAR(254), "network" VARCHAR(254), "cetSigs" VARCHAR(254), "fundingSigs" VARCHAR(254));
```

Comment:
was this created using a db dump after generating with Slick? If not just know that that is the best way to ensure that slick plays nice with our dbs :)

Reply:
I did not do that, I basically just copy pasted a previous `CREATE TABLE IF NOT EXISTS` and changed the column names and types